app/models/checkin.rb

```diff
@@ -42,6 +42,10 @@ class Checkin < ApplicationRecord
     end
   end
 
+  def self.import(file, device)
+    system "rake checkins:import[#{file.path},#{device}] &"
```

Comment:
Then, if we need a rake task to import checkins, we can just call service/interactor from this task.